pages/signup/register.tsx

```diff
@@ -1,60 +1,139 @@
 import React, { useState } from "react";
 import { Formik } from "formik";
-import { Button, TextInput, Label } from "@components/forms";
+import { Button, TextInput, Label, Alert, ErrorListItem } from "@components/forms";
 import { useAuth } from "@lib/hooks";
 import { useTranslation } from "next-i18next";
 import { GetServerSideProps } from "next";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import { Confirmation } from "@components/auth/Confirmation/Confirmation";
+import * as Yup from "Yup";
```

Comment:
Sorry just caught this. This import seems wrong here ? 

I would expect the import to look like 

```JS
import Yup from "yup"
```

Reply:
Oh right, yup will change yup :)